core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -241,7 +240,7 @@ protected void tryComplete() {
 
   protected synchronized void tryTerminate() {
     if (this.state == STATE_COMPLETE && (externalCompletion == null || externalCompletion.isDone())) {
-      if (LOGGER.isDebugEnabled()) {
+      if (debugLogEnabled) {
         LOGGER.debug(this + " terminated.");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use the built-in formatting to construct this argument. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLOTwXotIXnmsUkh19&open=AWkLOTwXotIXnmsUkh19)

[ISSUE_KEY]: AWkLOTwXotIXnmsUkh19